code/game/gamemodes/cult/runes.dm

```diff
@@ -945,6 +947,10 @@ var/list/teleport_runes = list()
 	notify_ghosts("Manifest rune created in [get_area(src)].", 'sound/effects/ghost2.ogg', source = src)
 
 /obj/effect/rune/manifest/can_invoke(mob/living/user)
+	if(is_busy)
+		to_chat(user,"<span class='cultitalic'>The rune is already in use.</span>")
```

Comment:
One rune per-tile? Wasn't there a thing where you can have multiple runes in one spot that hurt someone but grant you health/blood and if used too much or fast you get bloodlust? I like the idea of multiple runes on the same tile being used at once